Cargo.toml

```diff
@@ -45,3 +45,6 @@ stderrlog = "0.4.1"
 sysconf = ">=0.3.4"
 time = "0.1"
 tiny_http = "0.6"
+cashaccount-sys = { git = "https://github.com/dagurval/cashaccount-sys.git" }
```

Comment:
do you think we should add a `cashaccount-sys` to the BU github org.

Reply:
Yes, I'd like that